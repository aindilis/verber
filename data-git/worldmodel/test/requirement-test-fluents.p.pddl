(define
 (problem REQUIREMENT-TEST-FLUENTS)
 (:domain REQUIREMENT-TEST-FLUENTS)
 (:objects obj2 obj1 - type1)
 (:init
  (pred1 obj1))
 (:goal
  (and
   (pred1 obj2))))