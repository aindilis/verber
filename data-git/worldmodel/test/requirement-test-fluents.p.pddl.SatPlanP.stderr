
/var/lib/myfrdcsa/codebases/internal/verber/data/worldmodel/test/requirement-test-fluents.d.pddl: syntax error in line 3, ':FLUENTS':
requirement %s not supported by this IPP version :FLUENTS

Memory limit exceeded: bb system call, OR Time limit exceeded: bb system call, OR bb internal error
